code/modules/defenses/sentry.dm

```diff
@@ -219,29 +273,48 @@
 
 	if(omni_directional)
 		setDir(get_dir(src, A))
-
-	actual_fire(A)
+	for(var/i = 0; i < burst; i++)
```

Comment:
for(var/i in 0 to burst - 1)

Reply:
Nice one, not come across that in the byond code before